### ISSUE TITLE
My custom container deleted

### ISSUE THREAD

**Issue by khatskelevich** [kraaden/autocomplete#83]
`detach()` removes my custom container from parent. But `attach()` add it to `<body>` instead of parent

```
function detach(): void {
        const parent = container.parentNode;
        if (parent) {
            parent.removeChild(container);
        }
    }

function attach(): void {
        if (!container.parentNode) {
            doc.body.appendChild(container); // append to parent if it exists
        }
    }
```

**Comment by kylegoines** [comment]
i am also having this issue. if you define a container, on first detach it removes the container and then the autocomplete popup appends itself to the DOM at the highest level

**Comment by kylegoines** [comment]
@khatskelevich after looking into this for way too long haha i figured out a way you can fix this locally

on detach if oyu reconfigure it a little bit
```
    function detach() {
        console.log('detach happening')
        var parent = container.parentNode
        if (parent) {
            container.innerHTML = ''
        }
    }
    ```

**Comment by denis-taran** [comment]
@kylegoines @khatskelevich The issue is now fixed in the version `8.0.2`
